desktop-app/src/store/index.ts

```diff
@@ -148,6 +148,10 @@ const schema = {
         },
         default: {},
       },
+      onlyVisibleParts: {
+        type: 'boolean',
+        default: false,
```

Comment:
Can we make the default as `true`? So that the current behaviour stays the same unless specifically modified in the settings.